tensorflow/training/docker/2.13/py3/cu118/Dockerfile.gpu

```diff
@@ -0,0 +1,386 @@
+FROM nvidia/cuda:11.8.0-base-ubuntu20.04 AS base_image
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get autoremove -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+FROM base_image AS common
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# TensorFlow major.minor version
+ENV TF_VERSION=2.13
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+ENV KMP_AFFINITY=granularity=fine,compact,1,0
+ENV KMP_BLOCKTIME=1
+ENV KMP_SETTINGS=0
+ENV RDMAV_FORK_SAFE=1
+
+ARG PYTHON=python3.10
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.10.12
+
+ARG OPEN_MPI_PATH=/opt/amazon/openmpi
+ARG EFA_PATH=/opt/amazon/efa
+ARG EFA_VERSION=1.24.0
+ARG OMPI_VERSION=4.1.5
+ARG BRANCH_OFI=1.5.0-aws
+
+
+ARG CUDA=11.8
```

Comment:
cudnn/nccl version is the same as TF2.12?

Reply:
and efa

Reply:
cudnn is updated to 8.9.3 and EFA went to 1.24. NCCL did not change.